protocol/src/main/resources/protocol.xml

```diff
@@ -55,6 +55,7 @@
     <enum name="PartitionRole" encodingType="uint8">
       <validValue name="LEADER">0</validValue>
       <validValue name="FOLLOWER">1</validValue>
+      <validValue name="INACTIVE">2</validValue>
```

Comment:
How does this work on the gateway side? Would it work during rolling upgrade, gateway and broker is on different versions?

Reply:
A previous version would map the value to SBE_UNKNOWN but it shouldn't break